src/libs/SidebarUtils.js

```diff
@@ -230,7 +230,7 @@ function getOptionData(reportID) {
     };
 
     const participantPersonalDetailList = _.values(OptionsListUtils.getPersonalDetailsForLogins(report.participants, personalDetails));
-    const personalDetail = participantPersonalDetailList[0] || {};
+    const personalDetail = participantPersonalDetailList[0] || {login: ''};
```

Comment:
I don't particularly care for this solution. The reason is that when using default values in JS, we always use the absolute most simple default value. Please don't ask me why, I have no idea, it's just what is done and has been done for many many years, so let's keep it consistent! My guess is that you could go down some long rabbit hole of trying to provide a default value for every property that could exist on the `personalDetail` and it would become a mess. Once we have typescript, this problem will go away.

The way I would fix it is further down and make this change:

PREFERRED:
```
const login = Str.removeSMSDomain(personalDetail.login || '');
```

OR:
```
const login = personalDetail.login ? Str.removeSMSDomain(personalDetail.login) : '';
```

Reply:
>Once we have typescript, this problem will go away.

How? Because personal details will be a "type" with its defaults, so this won't even be a problem?

Reply:
I think how it will work is something like your IDE will throw an error (probably where the `.replace()` is happening) that `login` can be undefined and needs to be protected. This is because `personalDetail` will be a strictly defined type.